### DISCUSSION TITLE
CurrencyTextField from "@unicef/material-ui-currency-textfield" allows only 1 digit before decimal point

### DISCUSSION THREAD

**Discussion by keebeegee** [react-hook-form/react-hook-form#2639] [Q&A] [answered]
Using "CurrencyTextField ", I came across the following issue in V6.4.1 (and earlier):  
A User is not able to enter more than a total of 3 digits given decimalPlaces="2" 

[https://codesandbox.io/s/react-hook-form-controller-forked-mqpvw
](https://codesandbox.io/s/react-hook-form-controller-forked-mqpvw
)
![image](https://user-images.githubusercontent.com/7610366/90495639-22240380-e145-11ea-9a39-b2aba3342b2a.png)

Using as an native component all looks good: 
https://codesandbox.io/s/react-hook-form-controller-forked-oqpws

Your insight is greatly appreciated!

**Comment by bluebill1049** [comment, marked as answer]
Take a look at this codesandbox: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r

there is a mask input there for you to follow.

**Reply by keebeegee** [reply]
After some more looking-around I found this: 
https://codesandbox.io/s/react-hook-form-normalize-field-1jumn
linked form the example Github page: 
https://github.com/react-hook-form/react-hook-form/tree/master/examples
Should do the job - thanks for the quick response.

**Reply by bluebill1049** [reply]
good find! where did you find this reference?

**Reply by grazielleanna** [reply]
Hi!

How did you solve this problem? I'm managing to solve.
